### PR TITLE
add checker for tracking production store sas

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: options [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pattern_extractor="curl -sI https://assertions.ubuntu.com/_status/ping | grep Revision | perl -pe 's|X-Vcs-Revision: (\d+)|\1|'"
```
Comment (niemeyer) [inline options]: Same about perl. Also, with sed you can use `sed -n "..../p"` so only the correct line is printed.

Comment (niemeyer) [inline options]: (avoiding grep)

Let's please apply this to all the options files.

Comment (fgimenez) [inline options]: Done, thanks!

**Comment by fgimenez** [comment]
Merging this in its current status to unblock the test deployment, we can continue the discussion in further PRs.

**Pull request merged**